docs/AWS.md

```diff
@@ -0,0 +1,148 @@
+# Running calico-docker on AWS
+Calico runs on the Amazon Web Services (AWS), but there are a few tweaks required to the main Getting Started instructions.  The following instructions show the full power of the Calico routing and security model on AWS (and allow AWS to be used for testing).
+
+## Getting started
+These instructions describe how to set up three CoreOS hosts on AWS.  For more general background, see [the CoreOS on AWS documentation](https://coreos.com/docs/running-coreos/cloud-providers/ec2/).
+
+Download and install AWS Command Line Interface: 
+```
+curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+unzip awscli-bundle.zip
+sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+```
+For more information, see Amazon's [Installing the AWS Command Line Interface](http://docs.aws.amazon.com/cli/latest/userguide/installing.html#install-bundle-other-os).
+
+Configure the AWS CLI with your User keys.
+```
+aws configure
+
+  AWS Access Key ID: <User Access Key>
+  AWS Secret Access Key: <User Secret Access Key>
+  Default region name: us-west-2
+  Default output format: <json, text, or table>
+```
+Your AWS user needs to have the policy AmazonEC2FullAccess or be in a group with this policy in order to run the ec2 commands.  This can be set in the user configuration page of the web console.
+For more information on configuration and keys, see Amazon's [Configuring the AWS Command Line Interface](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html).
+
+## Setting up AWS networking
+The AWS machines will require a login by default.  Create a Key Pair instead to use when SSHing into the instances.  
+```
+aws ec2 create-key-pair --key-name calicokey --output text > calicokey.pem
+chmod 400 calicokey.pem
+```
+
+A Security Group is required on the instances to control allowed traffic.  Create a Security Group and allow any machine to SSH, but restrict all other traffic that is not within the Security Group.
```

Comment:
"Create a Security Group and allow any machine to SSH, but restrict all other traffic that is not within the Security Group." verges on a double negative.  How about "Create a security group that allows only SSH from the internet as well as all traffic between instances within the group."